protocol/x/clob/keeper/orders.go

```diff
@@ -107,12 +109,66 @@ func (k Keeper) PlaceShortTermOrder(
 	orderStatus types.OrderStatus,
 	err error,
 ) {
-	msg.Order.OrderId.MustBeShortTermOrder()
-
 	lib.AssertCheckTxMode(ctx)
 	nextBlockHeight := lib.MustConvertIntegerToUint32(ctx.BlockHeight() + 1)
 
-	return k.placeOrder(ctx, msg, nextBlockHeight, k.MemClob)
+	order := msg.GetOrder()
+	order.OrderId.MustBeShortTermOrder()
+	orderLabels := order.GetOrderLabels()
+
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), metrics.PlaceOrder, metrics.Latency)
+	defer func() {
+		telemetry.IncrCounterWithLabels(
+			[]string{types.ModuleName, metrics.PlaceOrder, metrics.Count},
```

Comment:
In the future, I think we may want to consider changing PlaceOrder to PlaceShortTermOrder. 

`dydxprotocol.clob_place_order_count.count` lends itself to thinking the order is all order types.